---
abstract: "Epigenetics is increasingly recognized as an important molecular mechanism underlying phenotypic variation. To study DNA methylation in ecological and evolutionary contexts, epiRADseq is a cost‐effective next‐generation sequencing (NGS) technique based on reduced representation sequencing of genomic regions surrounding non‐/methylated sites. EpiRADseq for genome‐wide methylation abundance and ddRADseq for genome‐wide single‐nucleotide polymorphism (SNP) genotyping follow very similar library and sequencing protocols, but to date these two types of dataset have been handled separately. Here we test the performance of using epiRADseq data to generate SNPs for population genomic analyses. We tested the robustness of using epiRADseq data for population genomics with two independent datasets: a newly generated single‐end dataset for the European whitefish Coregonus lavaretus, and a re‐analysis of publicly available, previously published paired‐end data on corals. Using standard bioinformatic pipelines with a reference genome and without (i.e. de novo catalogue loci), we compared the number of SNPs retained, population genetic summary statistics and population genetic structure between data drawn from ddRADseq and epiRADseq library preparations. We found that SNPs drawn from epiRADseq are similar in number to those drawn from ddRADseq, with 55%–83% of SNPs being identified by both methods. Genotyping error rate was <5% in both approaches. EpiRADseq‐specific allele dropout was low (~1%). For summary statistics, such as heterozygosity and nucleotide diversity, there is a strong correlation between methods (Spearman's rho > 0.88). Furthermore, identical patterns of population genetic structure were recovered using SNPs from epiRADseq and ddRADseq approaches. We show that SNPs obtained from epiRADseq are highly similar to those from ddRADseq and are equivalent for estimating genetic diversity and population structure. This finding is particularly relevant to researchers interested in genetics and epigenetics on the same individuals because using a single epigenomic approach to generate two datasets greatly reduces the time and financial costs compared to using these techniques separately. It also efficiently enables correction of epigenetic estimates with population genetic data. Many studies will benefit from a combinatorial approach with genetic and epigenetic markers and this demonstrates a single, efficient method to do so."
authors:
- __Marco Crotti__
- Colin Adams
- Kathryn Elmer
date: "2020-03-16T00:00:00Z"
doi: "https://doi.org/10.1111/2041-210X.13395"
featured: true
image:
  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/pLCdAaMFLTE)'
  focal_point: ""
  preview_only: false
projects:
- internal-project
publication: biorxiv
#publication_short: In *STC*
publication_types:
- "3"
publishDate: "2017-01-01T00:00:00Z"
summary: 
tags:
- Articles
title: Population genomic SNPs from epigenetic RADs
#url_code: '#'
#url_dataset: '#'
url_pdf: https://besjournals.onlinelibrary.wiley.com/doi/epdf/10.1111/2041-210X.13395
#url_poster: '#'
#url_project: '#'
#url_slides: '#'
#url_source: '#'
#url_video: '#'
---


Scripts available at my [github](https://github.com/marcocrotti/SNPs-EpiRAD).

